index_hash.txt

```diff
@@ -0,0 +1 @@
+77cd8123feef419d8090326b284110d664e447a2f92a3c06863aacdd5d865497
```

Comment:
index.json and index_hash.txt should not be in here.

Please check https://github.com/betaflight/firmware-presets?tab=readme-ov-file#tips-for-authors